src/core/utils/signMessages.tsx

```diff
@@ -18,14 +18,25 @@ export const getSigningRequestDisplayDetails = (
   try {
     switch (payload.method) {
       case 'personal_sign': {
-        let message = payload?.params?.[0] as string;
-        let address = payload?.params?.[1] as Address;
-        // While this is technically incorrect
-        // we'll support anyways for compatibility purposes
-        // since other wallets do it
-        if (isAddress(message)) {
```

Comment:
It took me a lil bit to figure out why this wouldn't work correctly. It turns out `Sign in to Fileverse` is actually a 42 character hex string that does conform to an address. I think a more simple modification here would be to only do the param flip when message is an address and address is not an address (so checking both params rather than just the first one)

Reply:
Would just use isAddress for that (rather than recreating)